backend/battles/views.py

```diff
@@ -43,6 +50,25 @@ def get_context_data(self, **kwargs):
 
         return context
 
+    def form_valid(self, form):
+        self.object = form.save()
+        battle = self.object.battle
+        creator = battle.user_creator
+        opponent = battle.user_opponent
+
+        if self.object.trainer == battle.user_creator:
```

Comment:
Thinking if here is the right place for this. Shouldn't you be using celery to send these emails?

Reply:
Yeah actually that change is made in the next trello card that asks to run battle asynchronously. [The changes were made here](https://github.com/pantoja/PokeBattle/blob/async-battle/backend/battles/views.py#L65) if you want to check out.

Reply:
ok!